lang/en/docs/accounts/service-levels.md

```diff
@@ -1,61 +1,61 @@
 # Service Levels
 
-Service levels define the types, qualities and quantities of services provided to the accounts within our platform. The features associated with using our platform which are affected by the opted Service Level consist of the following list.
+Service levels define the types, qualities and quantities of services provided to the accounts within our platform. 
+The features associated with using our platform which are affected by the opted Service Level consist of the following list.
 
-## Features
+| Features                         | Free                  | Pro            | Team           | Enterprise     |
+| :-------------                   | :-----------          | :------------- | :------------- | :------------- |
+| Disk Space[^1]                   | 10GB                  | 10GB           | 10GB           | 10GB           |
+| Dropbox Space                    | 1GB                   | 1GB            | 1GB            | 1GB            |
+| Private Data[^2]                 | -                     | +              | +              | +              |
+| Account Members[^3]              | 1                     | 1              | 2              | 5              |
+| Support Severity[^4]             | Normal                | High           | High           | Urgent         |
+| Entity Limits: Projects[^5]      | 1                     | 2              | 5              | 10             |
+| Entity Limits: Materials         | Unlimited             | Unlimited      | Unlimited      | Unlimited      |
+| Entity Limits: Workflows         | Unlimited             | Unlimited      | Unlimited      | Unlimited      |
+| Entity Limits: Jobs              | Unlimited             | Unlimited      | Unlimited      | Unlimited      |
```

Comment:
Either we should put the values instead of `Unlimited`, or we should remove these table entries entirely.

Reply:
Removed.